.github/workflows/pr.yml

```
@@ -94,12 +94,10 @@ jobs:
           echo "predict result directory missing" >&2 
           exit 1
         fi
-      
-        echo "result lines "$(wc -l preds_dmpnn/DMPNN_preds.csv)
-        if [ "$(cat preds_dmpnn/DMPNN_preds.csv | wc -l)" -lt "6" ]; then
-          echo "predict result should have at least 5 lines. But had only $(cat preds_dmpnn/DMPNN_preds.csv | wc -l)" >&2 
+        
```

````suggestion

        dfpl convert -f tests/data
       find tests/data
```

to see all the files that are there